test_conformance/api/test_kernel_arg_info.cpp

```diff
@@ -117,8 +117,17 @@ class KernelArgInfo {
 static std::string generate_argument(const KernelArgInfo& kernel_arg)
 {
     std::string ret;
-    std::string address_qualifier =
-        get_address_qualifier(kernel_arg.address_qualifier);
+
+    const bool is_image = strstr(kernel_arg.arg_type, "image")
+        || strstr(kernel_arg.arg_type, "sampler");
+    std::string address_qualifier = "";
+    // Image Objects are always allocated from the global address space so the
+    // qualifier should not be specified
+    if (!is_image)
```

Comment:
If `is_image` is true, can't `kernel_arg.address_qualifier` also be `CL_KERNEL_ARG_ADDRESS_LOCAL`, `CL_KERNEL_ARG_ADDRESS_CONSTANT`, or `CL_KERNEL_ARG_ADDRESS_PRIVATE`, here?

If so, then `run_scalar_vector_tests()` should have another `continue` statement added to avoid duplication.

Reply:
No, we set the address qualifier once for images to be global at the start on line 744.

Reply:
I'd missed that image tests are handled in `run_image_tests()`, and (obviously) not `run_scalar_vector_tests()`.